tests/raftstore/server.rs

```diff
@@ -245,7 +252,7 @@ impl Simulator for ServerCluster {
 }
 
 pub fn new_server_cluster(id: u64, count: usize) -> Cluster<ServerCluster> {
-    new_server_cluster_with_cfs(id, count, ALL_CFS)
+    new_server_cluster_with_cfs(id, count, &[])
```

Comment:
because Cluster create_engines will create ALL_CFS for kv, CF_DEFAULT for raft by default.